documentation/SOURCES.md

```diff
@@ -0,0 +1,46 @@
+# Source of packages installed
+
+## From APT
+
+- Ubuntu standard repositories
+  - [Streisand Common Packages](https://github.com/jlund/streisand/blob/master/playbooks/roles/common/vars/main.yml)
+  - [`bind`](https://github.com/jlund/streisand/blob/master/playbooks/roles/dnsmasq/tasks/main.yml)
+  - [`dnsmasq`](https://github.com/jlund/streisand/blob/master/playbooks/roles/dnsmasq/tasks/main.yml)
+  - [Libreswan Compilation Dependencies](https://github.com/jlund/streisand/blob/master/playbooks/roles/l2tp-ipsec/vars/main.yml)
+  - [`monit`](https://github.com/jlund/streisand/blob/master/playbooks/roles/monit/tasks/main.yml)
+  - [OpenConnect Dependencies](https://github.com/jlund/streisand/blob/master/playbooks/roles/openconnect/vars/main.yml)
+  - [Shadowsocks Dependencies](https://github.com/jlund/streisand/blob/master/playbooks/roles/shadowsocks/vars/main.yml)
+  - [`sslh`](https://github.com/jlund/streisand/blob/master/playbooks/roles/sslh/tasks/main.yml)
+  - [`stunnel4`](https://github.com/jlund/streisand/blob/master/playbooks/roles/stunnel/tasks/main.yml)
+  - [`tinyproxy`](https://github.com/jlund/streisand/blob/master/playbooks/roles/tinyproxy/tasks/main.yml)
+  - [`ufw`](https://github.com/jlund/streisand/blob/master/playbooks/roles/ufw/tasks/main.yml)
+- from 3rd party repositories:
+  - [`nginx` (from nginx.org)](https://github.com/jlund/streisand/blob/master/playbooks/roles/nginx/tasks/main.yml)
+  - [`obbfs4proxy` (from deb.torproject.org)](https://github.com/jlund/streisand/blob/master/playbooks/roles/tor-bridge/tasks/main.yml)
+  - [Wireguard Packages (from wireguard PPA)](https://github.com/jlund/streisand/blob/master/playbooks/roles/wireguard/tasks/install.yml)
+
+## Source
+- [Libreswan (from official cloudfront CDN mirror)](https://github.com/jlund/streisand/blob/master/playbooks/roles/l2tp-ipsec/vars/main.yml)

```

Comment:
I don't think this or the OpenConnect mirror are official in the sense that upstream runs them. They're mirrored by @jlund ad-hoc. I would maybe just leave out the `()` and say it's from source for now.